innopoints/views/product.py

```diff
@@ -88,6 +90,8 @@ def create_product():
     if db.session.query(duplicate.exists()).scalar():
         abort(400, {'message': 'A product with this name and type exists.'})
 
+    if len(new_product.varieties) == 0:
+        abort(400, {'message': 'Please provide at least one variety'})
```

Comment:
A period at the end of the message and a blank line afterwards would be nice. Also, this check of length, though very explicit in meaning, seems not Pythonic at all. I think the convention is to just `if not new_product.varieties`